backbone.js

```diff
@@ -1720,7 +1720,10 @@
     // decoded for comparison. `%25` should not be decoded since it may be part
     // of an encoded parameter.
     decodeFragment: function(fragment) {
-      return decodeURI(fragment.replace(/%25/g, '%2525'));
+      fragment = decodeURI(fragment.replace(/%25/g, '%2525'));
+      // If contains Korean then encode hashString - Korean character crash bug on Safari Browser :(
+      fragment = /[ㄱ-힣]/.test(fragment) ? encodeURIComponent(fragment) : fragment;
```

Comment:
Hi @jdalton
Thank for your reply :)
I saw your replay and reported it
https://bugs.webkit.org/show_bug.cgi?id=165597